### ISSUE TITLE
Add support for `bathena`

### ISSUE THREAD

**Issue by vincentmoroz** [openandroidinstaller-dev/openandroidinstaller#535]
### Is your device already requested?

- [X] I have searched the existing issues

### Brand

Motorola

### Name

Defy 2021

### Device code

Bathena

### Model

XT2083-8

### Device images

```Markdown
https://wiki.lineageos.org/devices/guamp/variant2/
```

### Do you own the device and would be willing to test the installer?

- [X] I own the device and am willing to test the installer on it.

### Additional context

```Markdown
This device is in the same model group (XT2083) as the Moto G9/G9 Play and Lenovo K12 Note. Have attempted to use the LineageOS GUAMP build but this has failed at the adb sideload stage.
```

**Comment by vincentmoroz** [comment]
I corrected the issue of sideloading, GUAMP image side loads but does not boot. Endless boot loop results.

**Comment by tsterbak** [comment]
Hard to tell how to fix that. I'm not an expert in building lineage os...

**Comment by anon1892** [comment]
If the device codename is not **exactly** the same as the codenames supported by the custom ROM & recovery you want to install, you may brick your phone (i.e. it will become as useful as a brick)

**Comment by vincentmoroz** [comment]
Agreed, no custom ROMs for Motorola Defy 2021 BATHENA. The closest ROM is for GUAMP (Moto G9 Play) which is also an XT2083, but not the same model as BATHENA. LineageOS ROM for Moto G9 covers XT2083 models 1,3, 4 and 6, Defy is 8. Internal components for both are the same except for a slightly different screen resolution which is perhaps why GUAMP ROM won't boot on BATHENA. I was hoping someone who understands how to build a ROM might take an interest in getting a ROM working for the Defy 2021 given the work done so far with the G9. I have the vendor blobs extracted and can test the build to help but lack the programming ability to DIY. The Defy 2021 is a bit of a niche phone being a rugged model, so other people's interest may be a stretch. 

I have used GSI images successfully, with Andy Yan's LineageOS lite one of the better working images. Works as a daily driver. Some of the GSI's have the Moto features as an option which is nifty, however, a real Lineage build with regular updates would be amazing.

**Comment by tsterbak** [comment]
Would be great to have an official LineageOS version as well as a twrp version. Maybe propose that in the lineageos reddit thread? :)
